Source/Applications/openHistorian/openHistorian/ServiceHost.cs

```diff
@@ -1131,7 +1127,7 @@ private bool PreventFailoverStartup()
                 }
             }
 
-            LogStatusMessage("Failover logic detected: no remaining active node.", UpdateType.Warning);
```

Comment:
If we remove all of these references can we also get rid of the `LogStatusMessage` function?